### ISSUE TITLE
prevent `innernet` binary changing interface when run on the `innernet-server` machine

### ISSUE THREAD

**Issue by mcginty** [tonarino/innernet#169]
Reported by @amarao that running `innernet` on the machine that is the coordinating server causes all kinds of issues with the interface, which we should outright prevent and put a warning message in the binary itself.

**Comment by amarao** [comment]
`innernet fetch`, to be precise.

**Comment by mcginty** [comment]
@amarao does that mean that there was both an `/etc/innernet-server/foo.conf` and an `/etc/innernet/foo.conf`? Otherwise, `innernet` should say that no interface config was found when running `innernet fetch`.

**Comment by amarao** [comment]
yes, I made both. I use innernet on innernet-server host to fetch peers and associations.
